test/reporters/json.spec.js

```diff
@@ -27,16 +27,16 @@ describe('json reporter', function () {
     }));
 
     runner.run(function (failureCount) {
-      failureCount.should.be.exactly(1);
-      runner.should.have.property('testResults');
-      runner.testResults.should.have.property('failures');
-      runner.testResults.failures.should.be.an.instanceOf(Array);
-      runner.testResults.failures.should.have.a.lengthOf(1);
+      expect(failureCount).to.be(1);
+      expect(runner).to.have.property('testResults');
+      expect(runner.testResults).to.have.property('failures');
+      expect(runner.testResults.failures).to.be.an('array');
+      expect(runner.testResults.failures.length).to.equal(1);
```

Comment:
Nit: whereas `should` has `.have.a.lengthOf`, the equivalent in `expect` is `.have.length`, which yields better error messages than `equal`... Would merge this either way, but figure it's worth pointing out.